source/core/NumericString.cpp

```diff
@@ -488,7 +488,7 @@ void Format(SubString *format, Int32 count, StaticTypeAndData arguments[], Strin
                             if (tempDouble < 0) {
                             absDouble = 0.0 - tempDouble;
                             }
-                            exponent = floor(log10(absDouble));
+                            exponent = static_cast<Int32>(floor(log10(absDouble)));
```

Comment:
I think this is the sort of usage that reads better as a constructor rather than a cast:
`exponent = Int32(floor(log10(absDouble)));`

Reply:
With modern C++, we could use uniform initialization (using braces instead of parentheses) by default. Where brace initialization throws compiler errors (due to narrowing), we could use C++ style casts. Is that acceptable to everyone?